lib/node_modules/@stdlib/blas/ext/base/gapxsum/benchmark/benchmark.js

```diff
@@ -21,13 +21,20 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
+var uniform = require( '@stdlib/random/array/uniform' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var pow = require( '@stdlib/math/base/special/pow' );
 var pkg = require( './../package.json' ).name;
 var gapxsum = require( './../lib/main.js' );
 
 
+// VARIABLES //
+
+var options = {
+	'dtype': 'float64'
```

Comment:
```suggestion
	'dtype': 'generic'
```

This matches the previous benchmarks.